config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1338,4 +1338,31 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: kubelet-credential-provider
-    description: "tests alpha feature kubelet image credential provider"
+    description: "tests feature kubelet image credential provider"
+- name: ci-kubernetes-e2e-gcp-credential-provider
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - args:
+        - --repo=k8s.io/kubernetes=master
+        - --timeout=240
+        - --root=/go/src
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --cluster=
+        - --env=ENABLE_AUTH_PROVIDER_GCP=true
```

Comment:
Do we need to set any feature gates? xref: 

```
# Also, it is required that DisableKubeletCloudCredentialProviders and KubeletCredentialProviders
# feature gates are set to true for kubelet to use external credential provider.
```

from https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh#L557-L558

Reply:
It looks like `DisableKubeletCloudCredentialProviders` defaults to false (https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L856) and `KubeletCredentialProviders` defaults to true (https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L912).

So sounds like only `DisableKubeletCloudCredentialProviders` needs to be enabled?

Reply:
+1 to setting `DisableKubeletCloudCredentialProviders` to `true`, this will ensure weren't not invoking the in-tree credential plugin.